runhouse/servers/cluster_servlet.py

```diff
@@ -321,9 +323,9 @@ async def aperiodic_cluster_checks(self):
                 )
                 self.logger.warning(
                     f"Temporarily increasing the interval between two consecutive status checks. "
-                    f"Next status check will be in {round(INCREASED_INTERVAL / 60, 2)} minutes. "
+                    f"Next status check will be in {round(INCREASED_INTERVAL / 60, 2)} hour(s). "
```

Comment:
is this intentional to change to hours?

Reply:
yea, because we divide `INCREASED_INTERVAL` by 60 => convert it from minutes to hours.